src/functTest/groovy/com/bmuschko/gradle/docker/DockerJavaApplicationPluginFunctionalTest.groovy

```diff
@@ -319,6 +319,26 @@ ADD file2.txt /other/dir/file2.txt
         build('help')
     }
 
+    def "Can enable configuration cache"() {
```

Comment:
Let's move this kind of test into a different class. It's somewhat random to use it for one of the convention plugins. I'd rather see the test here: `DockerfileFunctionalTest`.

Reply:
Makes sense

Reply:
Sounds good, I was able to move it over without requiring too much additional setup